DependencyInjection/DoctrineExtension.php

```diff
@@ -115,6 +115,12 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         foreach ($config['connections'] as $name => $connection) {
             $this->loadDbalConnection($name, $connection, $container);
         }
+
+        if (class_exists(Doctrine\Bundle\DoctrineBundle\Command\Proxy\PingCommand::class)) {
```

Comment:
This is never going to return true, as the namespace is relative to the current namespace. Even then, this would never return true as the class in question doesn't exist. You should import the correct class (`use Doctrine\DBAL\Tools\Console\Command\PingCommand;`) at the top of the file, then change this to:
```suggestion
        if (class_exists(PingCommand::class)) {
```

Reply:
Yes!